contracts/PolygonZkEVM.sol

```diff
@@ -409,14 +404,6 @@ contract PolygonZkEVM is OwnableUpgradeable, EmergencyManager {
         _;
     }
 
-    modifier isForceBatchAllowed() {
```

Comment:
`isForceBatchAllowed` should be added in the `Mock` version for testing purposes